deploy/firstboot/centos7-kvm.sh

```diff
@@ -93,3 +93,8 @@ STP=yes" > /etc/sysconfig/network-scripts/ifcfg-cloudbr1
 # Reboot
 reboot
 
+# Keep the script running until the reboot happens
+while :
+do
+  # loop infinitely
```

Comment:
That could be better, but I'm not really worried about what the VM is doing in between the time it reboots...